controllers/storagecluster/provider_server.go

```diff
@@ -196,7 +201,11 @@ func (o *ocsProviderServer) createService(r *StorageClusterReconciler, instance
 		}
 
 		instance.Status.StorageProviderEndpoint = fmt.Sprintf("%s:%d", endpoint, ocsProviderServicePort)
-	} else {
+
+	case corev1.ServiceTypeClusterIP:
+		instance.Status.StorageProviderEndpoint = fmt.Sprintf("%s:%d", actualService.Spec.ClusterIP, ocsProviderServicePort)
+
+	default:
```

Comment:
Can you pls add a comment here that the default value is Node port that's why we are adding it in default case and someone not change this.

Reply:
Done